app/components/dashboard/hypertension/overdue_patients_called_component.rb

```diff
@@ -28,11 +28,11 @@ def graph_data
     {
       overduePatients: data[:overdue_patients],
       overduePatientsCalled: data[:patients_called],
-      overduePatientsCalledRate: data[:patients_called_rates],
+      overduePatientsCalledRate: data[:patients_called_rates].map { |k, v| [k, cap_percentage(v)] }.to_h,
       startDate: @period.advance(months: -17),
-      calledWithResultAgreedToVisit: data[:patients_called_with_result_agreed_to_visit_rates],
-      calledWithResultRemindToCallLater: data[:patients_called_with_result_remind_to_call_later_rates],
-      calledWithResultRemoveFromOverdueList: data[:patients_called_with_result_removed_from_list_rates],
+      calledWithResultAgreedToVisit: cap_call_result_rates(:patients_called_with_result_agreed_to_visit_rates),
```

Comment:
This figure should never cross 100%?

Reply:
No, these are the rates! It gets capped proportionally only if the call rates > 100%

Reply:
If these are rates - they can never be over 100% because the denominator is the number of calls made? 
We cant have more people 'agree to visit' that calls made.

![Screenshot 2023-06-14 at 14 27 09](https://github.com/simpledotorg/simple-server/assets/9541902/a4512caa-9770-439a-b2d7-6ba14ce7330c)